### DISCUSSION TITLE
How to hard reset password?

### DISCUSSION THREAD

**Discussion by chantlong** [directus/directus#16792] [Archive]
My login email and password is not working anymore when I try to login a self hosted Directus.
I tried resetting my password by asking it to reset via the email I provided, which didn't work (No email came)
I am wondering how can I do a hard reset then. 
I tried passing in ADMIN_EMAIL , ADMIN_PASSWORD as env in a local startup of Directus but that didn't work either.

**Comment by br41nslug** [comment]
> I tried resetting my password by asking it to reset via the email I provided, which didn't work (No email came)

That is logical unless you've explicitly set up mailing services. You might want to look into a local mail testing tool to debug these like [MailDev](https://github.com/maildev/maildev), [MailHog](https://github.com/mailhog/MailHog) or [MailTrap](https://mailtrap.io/).

> I tried passing in ADMIN_EMAIL , ADMIN_PASSWORD as env in a local startup of Directus but that didn't work either.

These environment variables are only used to create a user on the initial installation and will indeed have no effect on a running instance.

What you probably want to do is create a new admin user so you can log in and see what went wrong 😄 https://docs.directus.io/reference/cli.html#creating-users

**Reply by chantlong** [reply]
Thanks a lot.
Initially I used the command line `npx directus users passwd --email abc --password abc` and mistakenly assumed it created a user.
I actually needed to use the `users create` command.
